lookup.go

```diff
@@ -33,11 +35,15 @@ func tryFormatLoggableKey(k string) (string, error) {
 		cstr = k
 	}
 
+	var encStr string
 	c, err := cid.Cast([]byte(cstr))
-	if err != nil {
-		return "", fmt.Errorf("loggableKey could not cast key to a CID: %x %v", k, err)
+	if err == nil {
+		encStr = c.String()
```

Comment:
Separating the CID check from the rest seems reasonable since it'll keep existing logging the same.

Is it weird that everything except sha256 multihashes are in base32, but that sha256 multihashes are in base58 (including if they were passed in as CIDv1s)?

Reply:
Meh.